src/sentry/workflow_engine/processors/delayed_workflow.py

```diff
@@ -71,61 +72,82 @@ def fetch_group_to_event_data(
     return buffer.backend.get_hash(model=model, field=field)
 
 
-def get_dcg_group_workflow_data(
-    worklow_event_dcg_data: dict[str, str]
-) -> tuple[dict[int, set[int]], dict[int, int], list[Any]]:
+def get_dcg_group_workflow_detector_data(
+    workflow_event_dcg_data: dict[str, str]
+) -> tuple[dict[int, set[int]], dict[int, int], dict[int, int], list[Any]]:
```

Comment:
this is a bit confusing with this many dicts, can we name some of these them? what do each of the dicts represent in the response?